playground/frontend/pubspec.lock

```diff
@@ -112,7 +112,7 @@ packages:
       name: built_value
       url: "https://pub.dartlang.org"
     source: hosted
-    version: "8.3.2"
+    version: "8.3.3"
```

Comment:
Is there a reason to update this file?
I usually only do this when `pubspec.yaml` changes for bulk updates. Otherwise it may get harder to merge for someone. For instance, there is now `v8.4.0` of this package. I like keeping version updates conscious and not accidental.